metadata-ingestion/docs/sources/business-glossary/datahub-business-glossary.md

```diff
@@ -36,6 +36,7 @@ The business glossary source file should be a `.yml` file with the following top
 - **inherits**: (optional) List of **GlossaryTerm** that this term inherits from
 - **contains**: (optional) List of **GlossaryTerm** that this term contains
 - **custom_properties**: A map of key/value pairs of arbitrary custom properties
+- **domains**: (optional) List of domain name or domain urn
```

Comment:
let's just make it singular for now to match the UI: just one domain at most

Reply:
Done